src/controllers/token-controller.js

```diff
@@ -1,23 +1,28 @@
+const joi = require('joi');
 const tokenService = require('../services/token-service');
 
+const token = joi.object({
+  mail: joi.string().email().max(100).required(),
+});
```

Comment:
why a token it has been validate with email?

Reply:
i tink i need to change de name of the var, in the request i only ask for de mail, so thts why is validate as mail